### PR TITLE
Re-enable nix based ci check

### PR DESCRIPTION
**Pull request by jappeace**

### CODE REVIEW

File: package.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name:                fakedata-quickcheck
-version:             0.2.0
+version:             0.1.0
```
Comment (psibi) [inline package.yaml]: We have a 0.2 release already made to hackage with minor documentation fixes.

File: package.yaml [hunk 2]
```diff
@@ -52,8 +52,8 @@ default-extensions:
 dependencies:
 - base >= 4.7 && < 5
 - QuickCheck >= 2.6 && < 2.15
-- fakedata
-- random
+- fakedata >= 0.6.0 && < 1.1.0
+- random  >= 1 && < 1.3.0
```
Comment (psibi) [inline package.yaml]: While I'm not opposed to this, having relaxed bounds makes maintaining things in Stackage easier. Also is there any reason on why we don't want to support older `random` package ?

Comment (jappeace) [reply package.yaml]: this isn't really my intention, I mucked up merging.

File: package.yaml (hunk 1 above)
Comment (jappeace) [reply package.yaml]: Got it, sorry about that.

File: package.yaml (hunk 2 above)
Comment (psibi) [reply package.yaml]: No worries, thank you!

File: fakedata-quickcheck.cabal [hunk 3]
```diff
@@ -1,6 +1,6 @@
 cabal-version: 1.12
 
--- This file has been generated from package.yaml by hpack version 0.34.4.
+-- This file has been generated from package.yaml by hpack version 0.34.2.
```
Comment (psibi) [inline fakedata-quickcheck.cabal]: Not exactly required: But do you think you can undo the cabal file changes too ?

Comment (jappeace) [reply fakedata-quickcheck.cabal]: You're right, no need to do unnecessary change.

**Comment by psibi** [comment]
Thank you!

**Pull request merged**